### PR TITLE
Update pt-PT.json

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

File: localization/languages/pt-PT.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json]: Can you undo this change?

File: localization/languages/pt-PT.json [hunk 2]
```diff
@@ -99,7 +99,8 @@
         },
         "settingsAppearanceHeading": "Aparência",
         "settingsDarkModeToggle": "Ativar modo escuro",
-        "settingsSwipeNavigationHeading": "Navegação com gestos",
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json]: Can you make this have the previous indentation again?

File: localization/languages/pt-PT.json [hunk 3]
```diff
@@ -148,5 +149,7 @@
         "appMenuShowAll": "Mostrar tudo",
         "appMenuQuit": "Sair do %n",
         "appMenuBringToFront": "Trazer tudo para a frente"
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json]: You'll need to add a comma at the end of this line for the JSON to be valid

**Comment by PalmerAL** [comment]
Thanks! I've added a couple of comments, otherwise this looks good.

**Comment by smarquespt** [comment]
Thanks. Done in https://github.com/minbrowser/min/pull/461/
